src/test/java/Home20work.java

```diff
@@ -1,30 +1,38 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.ElementClickInterceptedException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.time.Duration;
 
 
 public class Home20work {
+    static WebDriverWait wait=null;
+
     @Test
-    public void clearCurrentQueue() {
-        starup();
-        //login();
-    }
+    private static void registration1() {
```

Comment:
I recommend to use in the method naming:
-Follow code convention
-1 and 0 harcoded

AVOID
•	Magic numbers
•	Hungarian notation L
•	Abbreviations
•	x, y, z or any one-letter apart i/j
•	file2, sum1
•	Unclear "flag" "result" "status"

For example

`private static void registrationTest`